internal/storage/postgres/flowerbed/flowerbed.go

```diff
@@ -179,6 +179,14 @@ func (r *FlowerbedRepository) UnlinkAllImages(ctx context.Context, id int32) err
 }
 
 func (r *FlowerbedRepository) UnlinkSensorID(ctx context.Context, sensorID string) error {
+
+	if sensorID == "" {
+		return errors.New("sensorID cannot be empty")
+	}
+
+	if len(sensorID) > 0 && sensorID[0] == '-' {
```

Comment:
What are you checking here? Please keep in mind that the sensorID is a string, not an array.